src/utils/getPercentageOfDaysLeft.ts

```diff
@@ -0,0 +1,21 @@
+import moment from 'moment';
+
+function inputParser(input: string) {
+    const parsedDate = moment(new Date(parseInt(input, 10) * 1000));
+    return parsedDate;
+}
+
+const getPercentageOfDaysLeft = (startedOn: string, endsOn: string): number => {
+    const startDate = inputParser(startedOn);
```

Comment:
- Please use a consistent method of declaring functions.

Reply:
can you please let me  know what do you mean by it?